src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -139,12 +155,28 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
       // this.intervalId = setInterval(() => {
         this.updateUss(this.path);
       // }, this.updateInterval);
+    this.keyBindingSub.add(this.appKeyboard.keydownEvent
+      .subscribe((event) => {
+        if (event.which === KeyCode.KEY_Q && event.ctrlKey) {
```

Comment:
The header of the PR says about `Ctrl + Alt + P`. The code uses `Ctrl + Alt + Q`. Which one is correct?